### ISSUE TITLE
float16 quantized DistilBERT model results into performance drop

### ISSUE THREAD

**Issue by sayakpaul** [huggingface/tflite-android-transformers#11]
@Pierrci 

I used the DistilBERT model with the SST-2 dataset for text classification. I then converted the trained model to TensorFlow Lite using float16 quantization. Here's my [notebook](https://github.com/sayakpaul/BERT-for-Mobile/blob/master/DistilBERT_SST-2_TPU.ipynb). Then when I evaluated the float16 TensorFlow Lite model I see a tremendous performance drop (~49% validation accuracy) with respect to the original model. Here's the [notebook](https://github.com/sayakpaul/BERT-for-Mobile/blob/master/Evaluation_SST_2_DistilBERT.ipynb). 

Am I missing out on something?
